go-controller/pkg/ovn/gateway_cleanup.go

```diff
@@ -262,10 +262,11 @@ func (oc *Controller) removeLRPolicies(nodeName string, priorities []string) {
 }
 
 // removes DGP, snat_and_dnat entries, and LRPs
-func (oc *Controller) cleanupDGP(nodes *kapi.NodeList) error {
+func (oc *Controller) cleanupDGP(nodes []*kapi.Node) error {
 	klog.Infof("Removing DGP %v", nodes)
 	// remove dnat_snat entries as well as LRPs
-	for _, node := range nodes.Items {
+	for _, node := range nodes {
+		node := *node
```

Comment:
nit: not needed but I think it's not necessarily bad to have anyway. Just a heads up in case your preference is to remove it.